.github/workflows/main.yml

```diff
@@ -54,14 +54,16 @@ jobs:
         - build_php_table_backend_utils
     runs-on: ubuntu-latest
     if: |
-        contains(fromJson('["success"]'), needs.build_datatypes.result)
+        always()
+        && (contains(fromJson('["success", "skipped"]'), needs.build_datatypes.result) || contains(fromJson('["success", "skipped"]'), needs.build_php_table_backend_utils.result))
     steps:
       - name: Tests passed
-        run: echo "Tests passed" >> $GITHUB_STATE
+        run: echo "Tests passed"
```

Comment:
tady bys teoreticky mohl vytahnout tu podmínku z "if" a pokud je false, tak zavolat "exit 1", aby to failnulo, pokud některý z testů failnul. 

Ale úplně nerozumím tomu, proč tam máš ten "always()" to znamená, že to poběží i když jeden z těch testů failne. Kdyby tam nebyl, tak bych čekal, že se to nespustí = není "test passed" a tím pádem se nezavolá split, protože ten požaduje aby proběhl "tests passed"

Reply:
Always je tam pretože tŕň GitHub tak funguje, že aby ti to prešlo ďalej len pre jednu linku musí tam byt ten always

Reply:
Jak "jednu linku"?

Reply:
Jo, jasně - když některý z nich bude skipped, tak jakoby nebude v tom "needs" a nespustilo by se to...

Reply:
Skusal som este vytiahnut ten if a moc to nepomohlo skusal som a debug ze by mi to napovedalo ale nic som sa nedozvedel. Mozno by som to nechal tak ked to mame otestovane ze to funguje